[2023-12-17T11:19:11.206+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T11:19:11.208+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T11:19:11.343+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T11:19:11.347+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T11:19:11.419+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-17T11:19:11.438+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-17T11:19:11.439+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-17T11:19:11.465+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-14 04:00:00+00:00
[2023-12-17T11:19:11.472+0000] {standard_task_runner.py:57} INFO - Started process 2447 to run task
[2023-12-17T11:19:11.478+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-14T04:00:00+00:00', '--job-id', '410', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmp9m8jn47q']
[2023-12-17T11:19:11.480+0000] {standard_task_runner.py:85} INFO - Job 410: Subtask Create_Table
[2023-12-17T11:19:11.576+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T11:19:11.578+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T11:19:11.583+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [running]> on host 1a58977f5573
[2023-12-17T11:19:11.874+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-14T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-14T04:00:00+00:00'
[2023-12-17T11:19:11.895+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-17T11:19:13.169+0000] {airQuality.py:160} INFO - Table air_quality Already Exist
[2023-12-17T11:19:13.256+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-17T11:19:13.307+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231214T040000, start_date=20231217T111911, end_date=20231217T111913
[2023-12-17T11:19:13.397+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T11:19:13.463+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-17T12:18:21.025+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:21.028+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:21.098+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:21.103+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:21.234+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-17T12:18:21.282+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-17T12:18:21.283+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-17T12:18:21.351+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-14 04:00:00+00:00
[2023-12-17T12:18:21.363+0000] {standard_task_runner.py:57} INFO - Started process 4163 to run task
[2023-12-17T12:18:21.384+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-14T04:00:00+00:00', '--job-id', '622', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpyd176n73']
[2023-12-17T12:18:21.393+0000] {standard_task_runner.py:85} INFO - Job 622: Subtask Create_Table
[2023-12-17T12:18:21.587+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:21.591+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:21.597+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [running]> on host 1a58977f5573
[2023-12-17T12:18:21.884+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-14T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-14T04:00:00+00:00'
[2023-12-17T12:18:21.914+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-17T12:18:22.524+0000] {airQuality.py:161} INFO - Table air_quality Already Exist
[2023-12-17T12:18:22.560+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-17T12:18:22.652+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231214T040000, start_date=20231217T121821, end_date=20231217T121822
[2023-12-17T12:18:22.727+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T12:18:22.906+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-17T12:19:10.732+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:19:10.734+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:19:10.800+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:19:10.802+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:19:10.882+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-17T12:19:10.908+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-17T12:19:10.910+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-17T12:19:10.940+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-14 04:00:00+00:00
[2023-12-17T12:19:10.950+0000] {standard_task_runner.py:57} INFO - Started process 4603 to run task
[2023-12-17T12:19:10.957+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-14T04:00:00+00:00', '--job-id', '712', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmp1j2disj0']
[2023-12-17T12:19:10.960+0000] {standard_task_runner.py:85} INFO - Job 712: Subtask Create_Table
[2023-12-17T12:19:11.074+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:19:11.075+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:19:11.079+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [running]> on host 1a58977f5573
[2023-12-17T12:19:11.285+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-14T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-14T04:00:00+00:00'
[2023-12-17T12:19:11.309+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-17T12:19:12.235+0000] {airQuality.py:161} INFO - Table air_quality Already Exist
[2023-12-17T12:19:12.263+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-17T12:19:12.300+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231214T040000, start_date=20231217T121910, end_date=20231217T121912
[2023-12-17T12:19:12.341+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T12:19:12.389+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:52:03.273+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:03.279+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:03.425+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:03.426+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:03.572+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-28T16:52:03.620+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-28T16:52:03.631+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:52:03.747+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-14 04:00:00+00:00
[2023-12-28T16:52:03.767+0000] {standard_task_runner.py:57} INFO - Started process 303 to run task
[2023-12-28T16:52:03.822+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-14T04:00:00+00:00', '--job-id', '895', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpgj5lf59k']
[2023-12-28T16:52:03.837+0000] {standard_task_runner.py:85} INFO - Job 895: Subtask Create_Table
[2023-12-28T16:52:04.104+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:04.106+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:04.113+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:52:04.559+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-14T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-14T04:00:00+00:00'
[2023-12-28T16:52:04.612+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:52:05.129+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:52:05.226+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:52:05.397+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231214T040000, start_date=20231228T165203, end_date=20231228T165205
[2023-12-28T16:52:05.529+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:52:05.764+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:55:05.182+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:05.186+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:05.344+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:05.346+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:05.468+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-28T16:55:05.516+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [queued]>
[2023-12-28T16:55:05.524+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:55:05.587+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-14 04:00:00+00:00
[2023-12-28T16:55:05.597+0000] {standard_task_runner.py:57} INFO - Started process 1289 to run task
[2023-12-28T16:55:05.609+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-14T04:00:00+00:00', '--job-id', '1143', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmp41mjsdk1']
[2023-12-28T16:55:05.623+0000] {standard_task_runner.py:85} INFO - Job 1143: Subtask Create_Table
[2023-12-28T16:55:05.884+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:05.887+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-14T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:05.898+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-14T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:55:06.331+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-14T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-14T04:00:00+00:00'
[2023-12-28T16:55:06.404+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:55:10.690+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:55:10.802+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:55:10.877+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231214T040000, start_date=20231228T165505, end_date=20231228T165510
[2023-12-28T16:55:11.232+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:55:11.393+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
